databricks_cli/stack/api.py

```diff
@@ -175,8 +171,8 @@ def _deploy_resource(self, resource_config, resource_status=None, **kwargs):
         if resource_service == JOBS_SERVICE:
             click.echo('Deploying job "{}" with properties: \n{}'.format(resource_id, json.dumps(
                 resource_properties, indent=2, separators=(',', ': '))))
-            new_databricks_id, deploy_output = self._deploy_job(resource_properties,
-                                                                databricks_id)
+            new_databricks_id, _ = self._deploy_job(resource_properties,
```

Comment:
Is there a TODO somewhere to clean this up?

Reply:
done

Reply:
Given that `_deploy_job` is a private method, what is stopping you from changing the return in this PR?